zokrates_pest_ast/src/lib.rs

```diff
@@ -552,7 +552,32 @@ mod ast {
 
     #[derive(Debug, FromPest, PartialEq, Clone)]
     #[pest_ast(rule(Rule::constant))]
-    pub struct ConstantExpression<'ast> {
+    pub enum ConstantExpression<'ast> {
+        DecimalNumber(DecimalNumberExpression<'ast>),
+        BooleanConstant(BooleanConstantExpression<'ast>),
```

Comment:
Same here. Terminology-wise `DecimalNumber` and `BooleanConstant` seem to be different levels of abstraction. 
We could move to `FieldLiteral` and `BooleanLiteral`. Not sure it's worth the effort.

Reply:
I'll change to `BooleanLiteral` but keep `DecimalNumber` as at this point we could have different types of constants for the same type (say decimal and hex)